doc/developer-guides/hld/hv-rdt.rst

```diff
@@ -0,0 +1,54 @@
+.. _hv_rdt:
+
+RDT Allocation Feature Supported by Hypervisor
+##############################################
+
+The hypervisor allows to use RDT (Resource Director Technology) allocation features to optimize performances of VMs. There are 2 sub-features: CAT (Cache Allocation Technology) and MBA(Memory Bandwidth Allocation), CAT is for cache resources and MBA is for memory bandwidth resources. The RDT in hypervisor is integrated with VM-Configuration feature, the resources consumed by each VM can be configured and restricted at compile time.
```

Comment:
missed CDP (Code and Data Prioritization). Please put one sentence to explain why only CAT is supported for the moment.

Reply:
Will add "Code and Data Prioritization (CDP) is an extension of CAT. CDP is not supported by hyppervisor now, that will be explained latter. " here, and add "If CDP is enabled, the number of available CLOS is reduced to half, that is unwanted for ACRN so CDP will not be used." at "Introduction of CAT capabilities"